benchmark/benchresult/main.go

```diff
@@ -70,7 +70,7 @@ func timeChange(title int, val1, val2 time.Duration) string {
 func compareTwoMap(m1, m2 map[string]stats.BenchResults) {
 	for k2, v2 := range m2 {
 		if v1, ok := m1[k2]; ok {
-			changes := k2 + "-"
+			changes := k2
 			changes += fmt.Sprintf("%10s  %12s  %12s   %8s \n", "\nTitle", "\tBefore", "After", "\tPercentage")
```

Comment:
Logically, this would be more intuitive:
```go
changes := k2 + "\n"
changes += fmt.Sprintf("%10s......", "Title", .......)
```

or even put the `'k2\n'` in the `fmt.Sprintf` call.

Reply:
Done.